jigasi/rootfs/defaults/sip-communicator.properties

```diff
@@ -8,6 +8,9 @@
 {{ $XMPP_PORT := .Env.XMPP_PORT | default "5222" -}}
 {{ $XMPP_SERVER := .Env.XMPP_SERVER | default "xmpp.meet.jitsi" -}}
 {{ $XMPP_SERVERS := splitList "," $XMPP_SERVER -}}
+{{ $ENABLE_VOSK_TRANSCRIPTION := .Env.ENABLE_VOSK_TRANSCRIPTION | default "0" | toBool  -}}
+{{ $ENABLE_GCLOUD_TRANSCRIPTION := .Env.ENABLE_GCLOUD_TRANSCRIPTION | default "0" | toBool  -}}
+{{ $VOSK_SERVER := .Env.VOSK_SERVER}}
```

Comment:
This is unnecessary if you are not going to apply any transformation, you can use `.Env.VOSK_SERVER` directly.